source/manuals/programming-languages/python/python.html.md.erb

```diff
@@ -0,0 +1,67 @@
+---
+title: Writing Python at GDS
+---
+
+# <%= current_page.data.title %>
+
+
+### About this manual:
+
+This manual is designed to aid developers in writing Python code that is clear and consistent, within, and across,
+projects at GDS.
+
+
+### Guidance
+
+We follow [PEP8][], in some cases [PEP8][] doesn't express a view (e.g. on the usage of language
+features such as metaclasses) in these cases we defer to the [Google Python style guide][GPSG].
+We suggest using [PEP8][] and the [Google Python style guide][GPSG] (in order of preference) as references unless
+something is explicitly mentioned in the [GDS Python Style Guide][GDSPSG].
+
+
+### Additional rules
+
+The [GDS Python Style Guide][GDSPSG] is the place for detailing GDS specific rules/ exemptions to [PEP8][] or the
+[Google Python style guide][GPSG]. An example of which is allowing a line length of 120 instead of 80.
```

Comment:
This is probably pointlessly pedantic, but it's 79 characters in PEP8, not 80...

Reply:
Good catch, updated, nothing is too pedantic for a coding styleguide 😁